src/item_menu.c

```diff
@@ -526,9 +526,9 @@ void GoToBagMenu(u8 bagMenuType, u8 pocketId, void ( *postExitMenuMainCallback2)
             gUnknown_0203CE58.location = bagMenuType;
         if (postExitMenuMainCallback2)
             gUnknown_0203CE58.bagCallback = postExitMenuMainCallback2;
-        if (pocketId <= 4)
+        if (pocketId <= POCKETS_COUNT - 1)
```

Comment:
`if (pocketId < POCKETS_COUNT)`

Reply:
Hahaha, I was _hoping_ that would be equivalent, but I forgot to actually check.